go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/hashicorp/terraform
 
 go 1.24.0
 
+godebug winsymlink=0
```

Comment:
TIL there is `godebug` stanza allowed here, I always assumed it's environment variables only

Reply:
it was new feature in go1.23 so you can build a module with `GODEBUG` options by default, which is perfect timing for this particular go1.23 problem ;)